packages/core/src/mods/header/index.tsx

```diff
@@ -1,19 +1,60 @@
-import React from 'react';
+import React, { useState, useCallback } from 'react';
 
 import styles from './index.module.less';
 
 import { Graph } from '@antv/x6';
+import { Modal, message } from 'antd';
 import Guide from './guide';
 import Export from './export';
 import ImportDSL from './importDSL';
-import ConnectStatus from './connectStatus';
+import ConnectStatus, { Status } from './connectStatus';
+import Configuration from './configuration';
+import { localConnect } from '../../api';
 
 interface IProps {
   flowChart: Graph;
 }
 
-const Header: React.FC<IProps> = (props) => {
+const Header: React.FC<IProps> = (props: IProps) => {
   const { flowChart } = props;
+  const [projectName, setProjectName] = useState<string>('');
+  const [status, setStatus] = useState<Status>(Status.disconnected);
+
+  // network
+  const syncLocal = useCallback(() => {
+    return localConnect()
+      .then((res) => res.json())
+      .then((data = {}) => {
+        console.log(data);
```

Comment:
Here, we recommend you to remove your code of console.log. Because our custom console component intercepts the global console method, all the code here will cause trouble, resulting in the output of this data when execute the single node.

Reply:
done :)